modules/standard/IO.chpl

```diff
@@ -343,6 +343,7 @@ Some of these subclasses commonly used within the I/O implementation include:
  * :class:`OS.EofError` - the end of file was reached
  * :class:`OS.UnexpectedEofError` - a read or write only returned part of the requested data
  * :class:`OS.BadFormatError` - data read did not adhere to the requested format
+ * :class:`OS.InsufficientCapacityError` - a read or write operation required more memory than was available
```

Comment:
I'd probably say "more storage than was available" just because more memory than was available makes me think we ran out of system memory, but we ran out of storage in a particular array. Or else just go straight for an array example -- e.g. "a read operation required more storage than was available e.g., reading into an array to small to store all of the requested data"